src/style.css

```diff
@@ -1 +1,67 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
```

Comment:
The font-weight '300' is used for the 'Avenir-Book.ttf' font. Ensure that this weight is supported by the font file to avoid rendering issues.